### ISSUE TITLE
execute query-hotpluggable-cpus qmp command cause kata-runtime create block

### ISSUE THREAD

**Issue by woshijpf** [kata-containers/runtime#1363]
# Description of problem

Host environment:
host has 72 cpus:
```bash
# lscpu
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                72
On-line CPU(s) list:   0-71
Thread(s) per core:    2
Core(s) per socket:    18
Socket(s):             2
NUMA node(s):          2
Vendor ID:             GenuineIntel
CPU family:            6
Model:                 63
Model name:            Intel(R) Xeon(R) CPU E5-2699 v3 @ 2.30GHz
Stepping:              2
CPU MHz:               2299.820
BogoMIPS:              4596.08
Virtualization:        VT-x
L1d cache:             32K
L1i cache:             32K
L2 cache:              256K
L3 cache:              46080K
NUMA node0 CPU(s):     0-17,36-53
NUMA node1 CPU(s):     18-35,54-71
```

I execute the following docker command and since qemu start with 1 cpu which is specified in configuration.toml file by `default_vcpus` field, so kata-runtime create process will hotplug another 3 vcpus into sandbox.
```bash
docker run -tid --net none -m 1G --cpus 4 --restart always busybox:latest sh
```
however after executing above command, I get the error info in terminal:
```bash
OCI runtime create failed: container did not start before the specified timeout 120s for cmd(pid=51356): /var/lib/docker/runtimes/kata-runtime, [/var/lib/docker/runtimes/kata-runtime --root /var/run/docker/runtime-kata-runtime/moby --log /run/docker/containerd/daemon/io.containerd.runtime.v1.linux/moby/84f74611945d7fd284bebb411ef34278b29e1e1dd486205a6695f8e6f8461004/log.json --log-format json create --bundle /var/run/docker/containerd/daemon/io.containerd.runtime.v1.linux/moby/84f74611945d7fd284bebb411ef34278b29e1e1dd486205a6695f8e6f8461004 --pid-file /var/run/docker/containerd/daemon/io.containerd.runtime.v1.linux/moby/84f74611945d7fd284bebb411ef34278b29e1e1dd486205a6695f8e6f8461004/init.pid --console-socket /tmp/pty219958516/pty.sock 84f74611945d7fd284bebb411ef34278b29e1e1dd486205a6695f8e6f8461004]: : unknown
```

Then I find the error info in the log file:
```bash
2019-03-11T10:14:26.863284+08:00|info|kata-runtime[51358]|time="2019-03-11T10:14:26.859188083+08:00" level=info msg="{\"execute\":\"query-hotpluggable-cpus\"}" arch=amd64 command=create container=84f74611945d7fd284bebb411ef34278b29e1e1dd486205a6695f8e6f8461004 name=kata-runtime pid=51358 source=virtcontainers subsystem=qmp
2019-03-11T10:14:26.877376+08:00|info|kata-runtime[51358]|time="2019-03-11T10:14:26.861440787+08:00" level=info msg="{\"return\": [{\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 71}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 70}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 69}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 68}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 67}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 66}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 65}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 64}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 63}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 62}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 61}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 60}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 59}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 58}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 57}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 56}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 55}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 54}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 53}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 52}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 51}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 50}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 49}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 48}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 47}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 46}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 45}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 44}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 43}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 42}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 41}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 40}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 39}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 38}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 37}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 36}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 35}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 34}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 33}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 32}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 31}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 30}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 29}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 28}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 27}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 26}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 25}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 24}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 23}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 22}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 21}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 20}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 19}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 18}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 17}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 16}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 15}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 14}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 13}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 12}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 11}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 10}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 9}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 8}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 7}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 6}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 5}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 4}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 3}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 2}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 1}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 0}, \"vcpus-count\": 1, \"qom-path\": \"/machine/unattached/device[0]\", \"type\": \"host-x86_64-cpu\"}]}" arch=amd64 command=create container=84f74611945d7fd284bebb411ef34278b29e1e1dd486205a6695f8e6f8461004 name=kata-runtime pid=51358 source=virtcontainers subsystem=qmp
2019-03-11T10:14:26.878258+08:00|info|kata-runtime[51358]|time="2019-03-11T10:14:26.872330897+08:00" level=info msg="{\"timestamp\": {\"seconds\": 1552270466, \"microseconds\": 868075}, \"event\": \"RTC_CHANGE\", \"data\": {\"offset\": 0}}" arch=amd64 command=create container=84f74611945d7fd284bebb411ef34278b29e1e1dd486205a6695f8e6f8461004 name=kata-runtime pid=51358 source=virtcontainers subsystem=qmp
2019-03-11T10:14:26.878706+08:00|warning|kata-runtime[51358]|time="2019-03-11T10:14:26.872383598+08:00" level=warning msg="Unable to decode response [{\"timestamp\": {\"seconds\": 1552270466, \"microseconds\": 868075}, \"event\": \"RTC_CHANGE\", \"data\": {\"offset\": 0}}\r\ncpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 70}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 69}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 68}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 67}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 66}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 65}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 64}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 63}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 62}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 61}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 60}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 59}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 58}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 57}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 56}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 55}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 54}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 53}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 52}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 51}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 50}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 49}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 48}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 47}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 46}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 45}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 44}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 43}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 42}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 41}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 40}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 39}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 38}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 37}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 36}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 35}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 34}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 33}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 32}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 31}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 30}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 29}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 28}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 27}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 26}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 25}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 24}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 23}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 22}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 21}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 20}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 19}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 18}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 17}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 16}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 15}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 14}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 13}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 12}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 11}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 10}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 9}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 8}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 7}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 6}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 5}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 4}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 3}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 2}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 1}, \"vcpus-count\": 1, \"type\": \"host-x86_64-cpu\"}, {\"props\": {\"core-id\": 0, \"thread-id\": 0, \"socket-id\": 0}, \"vcpus-count\": 1, \"qom-path\": \"/machine/unattached/device[0]\", \"type\": \"host-x86_64-cpu\"}]}] from VM: invalid character 'c' after top-level value" arch=amd64 command=create container=84f74611945d7fd284bebb411ef34278b29e1e1dd486205a6695f8e6f8461004 name=kata-runtime pid=51358 source=virtcontainers subsystem=qmp
```

log info indicates qmp read the wrong return message which is mixed by RTC_CHANGE event, and this bug is caused by scanner.Bytes() reuse the same underlying array between two scan() function, the 
sequential scan() function write the read date into the same buf.

# Expected result

docker command execute successfully.

# Actual result

docker command failed and with above output and log.

---

**Comment by egernst** [comment]
@woshijpf is this still an issue?  Or resolved with https://github.com/intel/govmm/issues/88 ?

**Comment by woshijpf** [comment]
Yes, I fix this issue in intel/govmm#88, and I will revendor it into kata-runtime soon.

**Comment by devimc** [comment]
@woshijpf I'm revendoring it here https://github.com/kata-containers/runtime/pull/1498
